File: practicas/src/practica3/services/UserService.java
package practica3.services;

import practica3.model.User;

public interface UserService {

    public void addUser(String name, String id, String address, String phoneNumber, String email, WelcomeUser userType);

    public void updateUser(String name, String id, String address, String phoneNumber, String email, User user);

    public void deleteUser(User user);

    User findUser(String id);
}

File: ParcialAnalisis/Services/userServices/UserService.py
from abc import ABC, abstractmethod


class UserService(ABC):

    @abstractmethod
    def addUser(self, name, id, address, phoneNumber, email):
        pass

    @abstractmethod
    def updateUser(self, name, id, address, phoneNumber, email):
        pass

    @abstractmethod
    def deleteUser(self, user):
        pass

    @abstractmethod
    def findUser(self, id):
        pass
